project/.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -0,0 +1,7 @@
+---
+name: ❓ Support Question
```

Comment:
Usually users get the following answer to the support question:

> Hi, we try to keep Github issues for bug reports and feature requests only. If you have a question, please ask it on Stack Overflow or on #sonata on [the symfony-devs slack](https://symfony.com/slack-invite).

See https://github.com/sonata-project/SonataPageBundle/issues/985 for example. May be `Question.md` should be removed from PR?

Reply:
Check the content of the template 😄

Reply:
Hm, maybe, I don’t know really, what do rest of @sonata-project/contributors think?

Reply:
I read only one line first, yes! )) and wanted to delete comment ) But i imagined, that i have a question; I click `New issue`; I see a beautiful button `Getting started` near `Support question` header with ❓ and  💬 emojis. I will not read weird message about "we try to keep", i will delete it and ask my question on github, then i will get a comment from @sonata-project and my issue will be closed =)

Reply:
Maybe use ⛔ in the title then?

Reply:
⛔ NO support questions
use slack or stackoverflow.

?